playlist/playlist-ui.js

```diff
@@ -56,6 +57,34 @@ window.playlistUI = that = {
 			jqPlaylist.css( "width", w );
 		}
 		return that;
+	},
+	push: function( name ) {
+		var
+			jqPlaylistFile = jqPlaylistContent.children(),
+			jqNewFile = $( "<a>" )
+		;
+
+		jqPlaylistFile.removeClass( "selected" );
+
+		jqNewFile
+			.attr({
+				id: "item" + api.playlist.currId,
+				class: "item selected",
+			})
+			.text( name )
+			.click( ( function( selectId ) {
+				return function() {
+					jqPlaylistContent.children().removeClass( "selected" );
+					$( this ).addClass( "selected" );
```

Comment:
`$( this )` N'utilise pas le contrusteur de jQuery comme si c'etait gratuit :D
Ya pas une variable ici qui pourrait faire l'affaire?